src/nvim/tui/tui.c

```diff
@@ -2077,7 +2077,12 @@ static void augment_terminfo(TUIData *data, const char *term, long vte_version,
   data->unibi_ext.enable_extended_keys = unibi_find_ext_str(ut, "Eneks");
   data->unibi_ext.disable_extended_keys = unibi_find_ext_str(ut, "Dseks");
   if (data->unibi_ext.enable_extended_keys == -1) {
-    if (!kitty && (vte_version == 0 || vte_version >= 5400)) {
+    if (kitty) {
```

Comment:
We should explicitly check for support rather than relying on `TERM`: https://sw.kovidgoyal.net/kitty/keyboard-protocol/#detection-of-support-for-this-protocol